MACOSX_NOTES

```diff
@@ -19,6 +19,37 @@
 
 brew install git mysql exiftool libjpeg shared-mime-info openssl imagemagick findutils
 
+# If you haven't done so already, install a more recent version of Bash and set
+# it as the default. You can find your installed version like this:
+# bash --version
+
+# Apple only includes Bash 3.2 from 2007 in all versions of Mac OS X even now,
+# because Bash > 4.0 uses GPLv3, and they don't want to support that license.
+# MO's scripts use syntax that requires Bash >= 4.0.
+# Reference: https://medium.com/p/7138bd1066ba
+# https://stackoverflow.com/questions/6047648/associative-arrays-error-declare-a-invalid-option
+# To install the newest Bash alongside older versions:
+# brew install bash
+
+# now check paths of all versions installed
+# which -a bash
+# you should get something like this
+# /usr/local/bin/bash
+# /bin/bash
+# check those versions
+# /usr/local/bin/bash --version
+# /bin/bash --version
+# now check
+# bash --version
+# should be the new one.
+# So far, so good. Now you have make this version the default.
```

Comment:
This isn't exactly what's true on my system.  I believe the real key is what `/usr/bin/env bash --version` returns since that's what's in the shebang line of our scripts.  For me, `which bash` returns `/opt/homebrew/bin/bash` and this is the version that gets run by `/usr/bin/env bash`.

Reply:
Note: we have a couple scripts that still run from `#!/bin/bash`: `dump` and `strip_checkpoint`. I don't suppose anyone would run those locally, though.

Others do run from `#!/usr/bin/env bash`.